src/Form/FieldInput.js

```diff
@@ -214,6 +219,7 @@ FieldInput.defaultProps = {
   columnWidth: 12,
   handleEvent() {},
   value: "",
+  startValue: "",
```

Comment:
BUSTED! 👮  
I think that is a residue from an attempt to fix switching uncontrolled to a controlled input. Maybe it even partially works, I'll double check.